Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue7395.cs

```diff
@@ -0,0 +1,119 @@
+﻿using System;
+using System.Collections.Generic;
+using Xamarin.Forms.CustomAttributes;
+using Xamarin.Forms.Internals;
+
+#if UITEST
+using NUnit.Framework;
+using Xamarin.Forms.Core.UITests;
+using System.Linq;
+#endif
+
+namespace Xamarin.Forms.Controls.Issues
+{
+#if UITEST
+	[Category(UITestCategories.CollectionView)]
+#endif	
+ 	[Preserve(AllMembers = true)]
+	[Issue(IssueTracker.Github, 7395, "Changing ItemTemplate does not work as expected", PlatformAffected.Android)]
+	public class Issue7395 : TestContentPage
```

Comment:
Can you add instructions for future testers?

Reply:
Sure!. Added instructions:
![7395-instructions](https://user-images.githubusercontent.com/6755973/65609493-289b1100-dfb0-11e9-8332-1ce0a31fedb7.gif)

Reply:
@jsuarezruiz Can we give instructions to click those buttons as well? 

"Click Two Columns. If all cells render correctly with 2 columns, then click Three Columns. If all cells render correctly with 3 columns, then the test passes."

Reply:
Sure, added.

Reply:
Thanks